docusaurus/docs/iOS/swiftui/swiftui-cookbook/blocking-users.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Blocking Users
+description: How to block and unblock users
+---
+
+## Introduction
+
+Blocking users is an essential feature in a chat app because it enhances user safety and experience. It allows individuals to protect themselves from harassment, spam, and unwanted interactions. By giving users control over their interactions, it helps maintain privacy, reduces the risk of cyberbullying, and promotes a respectful community atmosphere. 
+
+As a result, some app stores require this functionality as part of their review process.
+
+## Stream Chat
+
+The Stream Chat SDK provides blocking and unblocking users out of the box, as well as listing them.
```

Comment:
I was thinking about the "out of the box" term.

Not sure if something like this is better.
`The Stream Chat SDK provides a way for blocking and unblocking users, as well as listing all of the blocked users.`